src/core/AutomationPattern.cpp

```diff
@@ -243,8 +243,9 @@ void AutomationPattern::removeValue( const MidiTime & _time,
 	cleanObjects();
 
 	MidiTime newTime = _quant_pos ?
-				Note::quantized( _time, quantization() ) :
-				_time;
+				Note::quantized( _time,
+					DefaultTicksPerTact / 64 ) : // finest quantization
```

Comment:
Hmm yes, the first commit does look better, cause we never want to limit the deletion of automation points really, at any quantization.